### DISCUSSION TITLE
AutoLoader - add customization options

### DISCUSSION THREAD

**Discussion by VolkerW** [markmap/markmap#183] [Q&A]
using the autoloader is was easy to display this map:
```
<div class="markmap">
# markmap
## autoloader
## transformer
## view
</div>
```
How can I customize this. I tried adding them to the code:
```
<div class="markmap">
duration: 100
colorFreezeLevel: 1
# markmap
## autoloader
## transformer
## view
</div>
```
However, these customization only show up as more nodes.

Any suggestion?

**Comment by gera2ld** [comment]
https://stackblitz.com/edit/markmap-autoloader?file=index.html
